cartridges/int_adyen_SFRA/package.json

```diff
@@ -5,7 +5,7 @@
   },
   "devDependencies": {
     "codeceptjs": "^3.0.0",
-    "testcafe": "^1.3.3",
+    "testcafe": "^2.0.0",
```

Comment:
we need to remember to delete the entire auto generated /cartridges folder before generating them again for the release

Reply:
yes good point. 
we should mention that in the release doc.